qemu/tests/cfg/test_vdpa_control_virtqueue.cfg

```diff
@@ -0,0 +1,14 @@
+- test_control_virtqueue:
```

Comment:
variant name should as the same as file name.

Reply:
I have updated, please help review it again, thanks a lot.